olive/cli/launcher.py

```diff
@@ -37,6 +38,7 @@ def get_cli_parser(called_as_console_script: bool = True) -> ArgumentParser:
     PerfTuningCommand.register_subcommand(commands_parser)
     ConfigureQualcommSDKCommand.register_subcommand(commands_parser)
     ManageAMLComputeCommand.register_subcommand(commands_parser)
+    QuantizeCommand.register_subcommand(commands_parser)
```

Comment:
maybe you could move this somewhere higher (35?) manage aml and cloud cache don't run workflows so this order feels a bit weird in the documentation.